acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/ibft/IbftMiningAcceptanceTest.java

```diff
@@ -43,28 +44,52 @@ public void shouldMineOnSingleNode() throws IOException {
 
   @Test
   public void shouldMineOnMultipleNodes() throws IOException {
-    final PantheonNode minerNode1 = pantheon.createIbftNode("miner1");
-    final PantheonNode minerNode2 = pantheon.createIbftNode("miner2");
-    final PantheonNode minerNode3 = pantheon.createIbftNode("miner3");
-    final PantheonNode minerNode4 = pantheon.createIbftNode("miner4");
-    cluster.start(minerNode1, minerNode2, minerNode3, minerNode4);
+    final PantheonNode validator1 = pantheon.createIbftNode("miner1");
+    final PantheonNode validator2 = pantheon.createIbftNode("miner2");
+    final PantheonNode validator3 = pantheon.createIbftNode("miner3");
+    final PantheonNode nonValidatorNode = pantheon.createIbftNode("miner4");
```

Comment:
textual name (miner4) doesn't line up with variable name (nonValidatorNode) - which should it be?